cmd/agent/main.go

```diff
@@ -91,6 +91,26 @@ See https://docs.c3os.io/after_install/upgrades/#manual for documentation.
 		},
 	},
 
+	{
+		Name:      "notify",
+		Usage:     "notify <event> <config dir>...",
+		UsageText: "emits the given event with a generic event payload",
+		Description: `
+Sends a generic event payload with the configuration found in the scanned directories.
+`,
+		Aliases: []string{},
+		Flags:   []cli.Flag{},
+		Action: func(c *cli.Context) error {
+			dirs := []string{"/oem", "/usr/local/cloud-config"}
+			args := c.Args()
+			if len(args) > 1 {
+				dirs = args[1:]
+			}
+
+			return agent.Notify(args[0], dirs)
```

Comment:
small nit: we should consider failing here if we pass an event which is not exposed in the bus. in that way we can signal the user earlier if no event is registered in the sdk's bus. otherwise the event would be silently ignored

Reply:
I agree.  I think we should address error handling for plugins more broadly and make that change as part of that effort.